gravitee-apim-console-webui/src/management/api/portal/userGroupAccess/api-portal-user-group-access.module.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { GioIconsModule, GioSaveBarModule } from '@gravitee/ui-particles-angular';
+import { MatSelectModule } from '@angular/material/select';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatCardModule } from '@angular/material/card';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+
+import { ApiPortalAccessGroupsComponent } from './groups/api-portal-access-groups.component';
+
+import { GioPermissionModule } from '../../../../shared/components/gio-permission/gio-permission.module';
+
+@NgModule({
+  declarations: [ApiPortalAccessGroupsComponent],
+  exports: [ApiPortalAccessGroupsComponent],
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+
+    MatSelectModule,
+    MatCardModule,
+    MatSnackBarModule,
+
+    GioIconsModule,
+    GioPermissionModule,
+    GioSaveBarModule,
+  ],
+})
+export class ApiPortalUserGroupAccessModule {}
```

Comment:
WDYT to add @loriepisicchio screen here too? (I'm sure you're in sync but if in doubt)

Reply:
The UserGroupAccessModule is for all the pages in "userGroupAccess".